Documentation/gettingstarted.rst

```diff
@@ -51,25 +51,19 @@ repository:
 Step 2: Starting the Docker + Cilium VM
 ---------------------------------------
 
-Open a terminal and navigate into the top of the cilium source directory.
```

Comment:
The visual of a console font helps me to see what are the commands I need to run without reading the whole paragraph. I like better the way it look.